app/components/EmbedMarketModal.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import {
+  Dialog,
+  DialogBody,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  IconButton,
+} from '@swapr/ui';
+import { ModalId, useModal } from '@/context/ModalContext';
+import { APP_URL } from '@/constants';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { atomDark } from 'react-syntax-highlighter/dist/esm/styles/prism';
+import { EmbedMarketCard } from '@/app/components/EmbedMarketCard';
+import { useState } from 'react';
+interface EmbedMarketModalProps {
+  id: string;
+  fixedProductMarketMaker: any;
+}
+
+export const EmbedMarketModal = ({
+  id,
+  fixedProductMarketMaker,
+}: EmbedMarketModalProps) => {
+  const { isModalOpen, closeModal } = useModal();
+  const [clipboardIcon, setClipboardIcon] = useState<'copy' | 'tick'>('copy');
+
+  const close = () => {
+    closeModal(ModalId.EMBED_MARKET);
+  };
+
+  const iframeCode = `<iframe
+  title="presagio-market-iframe"
+  src="${APP_URL}/embed/market?id=${id}"
+  width="384"
+  height="222"
+></iframe>`;
+
+  return (
+    <Dialog open={isModalOpen(ModalId.EMBED_MARKET)} onOpenChange={close}>
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Embed Market</DialogTitle>
+        </DialogHeader>
+        <DialogBody className="space-y-6 px-6 pb-8">
+          <div className="space-y-2">
+            <h3 className="font-bold">Embed Code</h3>
+            <div className="relative rounded-16">
+              <div className="absolute right-2 top-2 z-10">
+                <IconButton
+                  onClick={() => {
+                    navigator.clipboard.writeText(iframeCode);
+                    setClipboardIcon('tick');
+
+                    setTimeout(() => {
+                      setClipboardIcon('copy');
+                    }, 3000);
+                  }}
+                  variant="pastel"
+                  name={clipboardIcon}
+                  size="sm"
+                />
+              </div>
+              <SyntaxHighlighter language="javascript" style={atomDark}>
+                {iframeCode}
+              </SyntaxHighlighter>
+            </div>
+          </div>
+          <div className="space-y-2">
```

Comment:
I think we should give a bit more space between the subtitle and the component. Top spacing and bottom spacing feel awkward. The same spacing to Embed Code too.

<img width="569" alt="image" src="https://github.com/user-attachments/assets/d417082c-bd4d-41a5-8c09-6af07e2e6b8b">

Reply:
Added more space.